### ISSUE TITLE
[Feature request] Allow referencing or copying of existing production sheets in another sheet

### ISSUE THREAD

**Issue by airfighter001** [ShadowTheAge/yafc#227]
Especially in larger mod packs, there are many production lines that require similar sub-lines again and again. So whenever you create a new production line, you'll have to click through these sub-lines again.
Being able to create production sheets with these sub-lines to then import them into your real production sheets whenever you need them would be really nice QoL.

Implementing that with a nice interface, dropdown menus, and stuff would be quite a bit of work I assume, so I came up with another way to implement it.
There's already a way to export a production sheet as a string to share it. What if you could use that string to import a sheet into another sheet as a nested table? I assume that would be easier as pretty much all of this already exists, just as a new production sheet, not a nested table in an already existing sheet.

Tl;dr: Add a button 'Import as nested table' that lets you import a production sheet string as nested table into an existing production sheet.
